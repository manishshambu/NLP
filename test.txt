1	start
2	over
3	.

1	a
2	moderate
3	distance
4	away
5	.

1	i
2	want
3	to
4	be
5	far
6	away
7	from
8	icsi
9	.

1	i
2	want
3	to
4	have
5	dinner
6	.

1	tell
2	me
3	about
4	le
5	bateau
6	ivre
7	.

1	it
2	can
3	be
4	really
5	expensive
6	.

1	i
2	want
3	to
4	eat
5	breakfast
6	at
7	a
8	really
9	cheap
10	place
11	tomorrow
12	.

1	i
2	want
3	to
4	eat
5	breakfast
6	at
7	a
8	really
9	cheap
10	place
11	tomorrow
12	.

