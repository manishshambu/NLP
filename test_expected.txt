1	start	VB
2	over	RP
3	.	.

1	a	DT
2	moderate	JJ
3	distance	NN
4	away	RB
5	.	.

1	i	PRP
2	want	VBP
3	to	TO
4	be	VB
5	far	RB
6	away	RB
7	from	IN
8	icsi	NN
9	.	.

1	i	PRP
2	want	VBP
3	to	TO
4	have	VB
5	dinner	NN
6	.	.

1	tell	VB
2	me	PRP
3	about	IN
4	le	FW
5	bateau	FW
6	ivre	FW
7	.	.

1	it	PRP
2	can	MD
3	be	VB
4	really	RB
5	expensive	JJ
6	.	.

1	i	PRP
2	want	VBP
3	to	TO
4	eat	VB
5	breakfast	NN
6	at	IN
7	a	DT
8	really	RB
9	cheap	JJ
10	place	NN
11	tomorrow	NN
12	.	.

1	i	PRP
2	want	VBP
3	to	TO
4	eat	VB
5	breakfast	NN
6	at	IN
7	a	DT
8	really	RB
9	cheap	JJ
10	place	NN
11	tomorrow	NN
12	.	.

